airgun/entities/contentview_new.py

```diff
@@ -23,25 +24,91 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def publish(self, entity_name, values=None):
+        """Publishes to create new version of CV and promotes the contents to
+        'Library' environment.
+        :return: dict with new content view version table row; contains keys
+            like 'Version', 'Status', 'Environments' etc.
+        """
+        view = self.navigate_to(self, 'Publish', entity_name=entity_name)
+        if values:
+            view.fill(values)
+        view.next.click()
+        view.finish.click()
+        view.progressbar.wait_for_result()
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.versions.table.read()
+
+    def check_if_blank_in_french(self):
+        view = self.navigate_to(self, 'French')
+        return view.table.read()
 
 @navigator.register(NewContentViewEntity, 'All')
 class ShowAllContentViewsScreen(NavigateStep):
     """Navigate to All Content Views screen."""
 
-    VIEW = NewContentViewTableView
+    VIEW = ContentViewTableView
 
     @retry_navigation
     def step(self, *args, **kwargs):
         self.view.menu.select('Content', 'Lifecycle', 'Content Views')
 
 
+@navigator.register(NewContentViewEntity, 'French')
```

Comment:
Here's the actual view introduced by this PR. This simply uses some different language to allow navigation to the CV page. ( Contenu instead of Content )